feconf.py

```diff
@@ -75,6 +75,11 @@
     'TextInput': 'LDAStringClassifier'
 }
 
+# A mapping of classifier algorithms to classifier schema data versions.
+CLASSIFIER_VERSION_MAPPING = {
```

Comment:
Done. A small query though : 
```
    algorithm_id = feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']['classifier_id']
``` 
How would I write the above line in less than 80 characters? @seanlip